language-support/ts/codegen/tests/ts/generated/src/__tests__/test.ts

```diff
@@ -79,6 +80,17 @@ test('create + fetch', async () => {
   expect(personContracts).toHaveLength(1);
   expect(personContracts[0]).toEqual(aliceContract);
 
+  // Alice has a birthday.
+  const [er, es] = await ledger.exercise(Main.Person.Birthday, aliceContract.contractId, {});
+  // Resulting in her old record being archived and replaced with a new one.
+  expect(es).toHaveLength(2);
+  const aliceOldContract: ArchiveEvent<Main.Person> = (es[0] as { archived: ArchiveEvent<Main.Person> }).archived;
+  const aliceNewContract: CreateEvent<Main.Person>  = (es[1] as { created: CreateEvent<Main.Person> }).created;
+  // The result of the exercise ('er') is her new record ID.
+  expect (er).not.toEqual(aliceContract.contractId);
```

Comment:
`expect` is a normal function.
```suggestion
  expect(er).not.toEqual(aliceContract.contractId);
```